src/toil/options/common.py

```diff
@@ -466,7 +466,8 @@ def is_within(x: Union[int, float]) -> bool:
     )
 
     caching = file_store_options.add_mutually_exclusive_group()
-    caching_help = "Enable or disable caching for your workflow, specifying this overrides default from job store"
+    caching_help = ("Enable or disable worker level file caching for your workflow, specifying this overrides default from batch system. "
```

Comment:
I think this is running two sentences together with a comma, which should really be `;` or `.`.